diff --git a/roboticstoolbox/mobile/DistanceTransformPlanner.py b/roboticstoolbox/mobile/DistanceTransformPlanner.py
index d14995f5..e937be78 100644
--- a/roboticstoolbox/mobile/DistanceTransformPlanner.py
+++ b/roboticstoolbox/mobile/DistanceTransformPlanner.py
@@ -2,7 +2,6 @@
 @Author: Peter Corke, original MATLAB code and Python version
 @Author: Kristian Gibson, initial MATLAB port
 """
-from numpy import disp
 from scipy import integrate
 from spatialmath.base.transforms2d import *
 from spatialmath.base.vectors import *
 
